modules/ROOT/pages/coverage-maven-concept.adoc

```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-partial:
+:page-deployment-options: acb-ide, studio-ide
 
 // tag::coverage-report[]
 
-Since MUnit 2.x, the coverage report goal is integrated with the maven reporting section. Coverage Reports are generated during Maven's `site` lifecycle, during the `coverage-report` goal.
+Since MUnit 2.x, the coverage report goal is integrated with the Maven reporting section. MUnit generates Coverage Reports during the Maven `site` lifecycle, during the `coverage-report` goal.
```

Comment:
Not sure if we use "Since" in this way or not because it has 2 potential meanings, the one you are using and also "because". Anyway, not sure if we have a style policy on this word. 

Sometimes, I write the wordy, "Versions 2.x and above integrate the coverage report into the Maven reporting section... " Or something. But again, not sure if we care about "Since"

Reply:
Why is it "Coverage Reports" in some cases and "coverage report" in others?

Reply:
Thanks for mentioning the "since". Reworded it! ;) 
And it should be "coverage report" (just a report) and "MUnit Coverage" (an MUnit feature).
Thanks!